README.md

```diff
@@ -22,7 +22,7 @@ for everyday use. It is aiming for power users with a modern feature mindset.
 - ✅ High-DPI support.
 - ✅ Vertical Line Markers (quickly jump to markers in your history!)
 - ✅ Vi-like input modes for improved selection and copy'n'paste experience and Vi-like `scrolloff` feature.
-- ✅ Blurred behind transparent background when using Windows 10 or KDE window manager on Linux.
+- ✅ Blurred behind transparent background support for Windows 10 and above as well as the KDE and GNOME desktop environment on Linux.
```

Comment:
What is "Blurred behind transparent background support"? Is it meant to be "Transparent and translucent (blurred) background support"?

Also presumably it is for KDE and GNOME on all platforms? Or is it really Linux specific?

Reply:
Users have the ability to control the transparency of their windows through a compositor, such as Picom or the standard ones for KDE and GNOME. 
This feature adds a blur effect to the background image visible when looking through a transparent window, creating following:
Image from the web:
![image](https://github.com/contour-terminal/contour/assets/44506630/fc84d173-89ab-42df-82c2-ccb130ecb2b4)

Reply:
Looks like we want to add a little video demo to https://contour-terminal.org/features/ as a sub-page, to illustrate this, maybe? 🤔

Reply:
Transparent and blurred backgrounds when run under Windows 10 (or above), GNOME or KDE, or with other compatible compositors.

?